CMSgov/mint-app
# [NOREF] Front End Implementation of Combining Milestones and Basics

# NOREF

## Changes and Description

- Changed  `UpdateModelPlan` mutation query to receive two calls: `updateModelPlan` and `updatePlanBasics`
- Updated `planBasics` validation schema to reflect the new structure 
- Updated `renderBasicsStatus` and `renderBasicsLastUpdated` in `src/views/ModelPlan/TaskList/index.tsx`


## How to test this change

1. Create new model plan
2. Navigate through Model Basics task item
3. Click through and ensure no errors and changes are properly recorded 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
